contrib/templates/experimental/scala-job/template/README.md.tmpl

```diff
@@ -0,0 +1,9 @@
+# {{.project_name}}
```

Comment:
Could you extend this README a bit to include some of hte standard boilerplate we have in https://github.com/databricks/cli/blob/main/libs/template/templates/default-sql/template/%7B%7B.project_name%7D%7D/README.md.tmpl and https://github.com/databricks/cli/blob/main/libs/template/templates/default-python/template/%7B%7B.project_name%7D%7D/README.md.tmpl?